File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/RaceByNameResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class RaceByNameResponse : IResponse
    {
        private readonly Race race;

        public RaceByNameResponse(Race race)
        {
            this.race = race;
        }

        public Race Race { get { return race; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab10/networking/src/main/java/com/ubb/protocol/response/RaceByNameResponse.java
package com.ubb.protocol.response;

import com.ubb.model.Race;

public class RaceByNameResponse implements Response {
    private final Race race;

    public RaceByNameResponse(Race race) {
        this.race = race;
    }

    public Race getRace() {
        return race;
    }
}
